pkg/network/ovn_kubernetes.go

```diff
@@ -276,14 +276,40 @@ func bootstrapOVN(kubeClient client.Client) (*bootstrap.BootstrapResult, error)
 
 	sort.Strings(ovnMasterIPs)
 
+	var clusterInitiator string
```

Comment:
A few sentences about what this does would be good. Something about picking a stable raft "join point"

Reply:
The const `OVNRaftClusterInitiator` has a doc bloc which explains this, but I could add a minor here as well

Reply:
thanks!